torch/onnx/symbolic_opset7.py

```diff
@@ -0,0 +1,11 @@
+from torch.onnx.symbolic_helper import _black_list_in_opset
+from torch.onnx.symbolic_opset9 import _adaptive_pool, _pair, _single, _triple, max_pool1d, max_pool2d, max_pool3d
+
+black_listed_operators = ["scan", "expand"]
```

Comment:
the right way to add support for opset 7 and opset 8 is putting all the operators in the black_list_operators... Let's don't import anything from opset 9, since it's very tricky to get it correct this way.